.ci.yaml

```diff
@@ -1126,6 +1126,23 @@ targets:
         ["devicelab", "hostonly", "linux"]
       task_name: linux_desktop_impeller
 
+  - name: Linux_android_emu android_display_cutout
+    recipe: devicelab/devicelab_drone
+    timeout: 60
+    presubmit: true
+    bringup: true
```

Comment:
Please double check these values. My intent is to land this pr and see if the tests pass then debug it on a follow up pr.

Reply:
So this isn't possible, as a bringup build means "do not run on presubmit". If it does work (I would expect a CI validation error), let me know... because I'm so surprised.

The suggestion that Jonah and others gave was to change an _existing_ pre-submit test to run your test instead, without submitting the .ci.yaml changes (i.e. keeping it as a draft or whatever floats your boat).

Reply:
So this isn't possible, as a bringup build means "do not run on presubmit". If it does work (I would expect a CI validation error), let me know... because I'm so surprised.

The suggestion that Jonah and others gave was to change an _existing_ pre-submit test to run your test instead, without submitting the .ci.yaml changes (i.e. keeping it as a draft or whatever floats your boat).

Reply:
FWIW .ci.yaml validation passed https://github.com/flutter/flutter/pull/162641/checks?check_run_id=36619533225
https://ci.chromium.org/ui/p/flutter/builders/prod/Linux%20ci_yaml%20flutter%20roller/17461/overview

I thought bringup meant "do not block the build if this test fails" which should be independent of "when to run" but our ci system is difficult to reason about. 

Tomorrow I will work on a version that modifies an existing test to see what happens.

Reply:
Yeah, if you see below it wasn't actually executed.

We should disallow this pattern for that reason: https://github.com/flutter/flutter/issues/162651.